data/mods/ssb/conditions.ts

```diff
@@ -1194,6 +1206,42 @@ export const Conditions: {[k: string]: ModdedConditionData} = {
 			}
 		},
 	},
+	// Brilliant Condition for Arcticblast
+	brilliant: {
+		name: 'Brilliant',
+		duration: 5,
+		onStart(pokemon) {
+			pokemon.addVolatile("ingrain");
+		},
```

Comment:
Just add the contents of the condition onto the mon

Reply:
I think adding ingrain will give the "Dug in roots" message